### DISCUSSION TITLE
stripe payment with vue-storefront

### DISCUSSION THREAD

**Discussion by macrozone** [vendurehq/vendure#1507] [Q&A]
Did anyone successfully integrate stripe into vendure with the vue-storefront? 

Is there some documentation, tutorial or similar?

**Comment by macrozone** [comment]
actually there was a piece missing in the vue-storefront vendure plugin: https://github.com/vuestorefront-community/vendure/pull/192

**Reply by michaelbromley** [reply]
ok, thanks for the follow-up and the PR!
